orchagent/flexcounterorch.cpp

```diff
@@ -244,6 +248,10 @@ void FlexCounterOrch::doTask(Consumer &consumer)
                     {
                         dash_orch->handleFCStatusUpdate((value == "enable"));
                     }
+                    if (dash_meter_orch && (key == DASH_METER_KEY))
```

Comment:
Is a new entry to FC table being added? If so, please also attach the buildimage and utilities PR (if any CLI is also being added)